server/server.js

```diff
@@ -0,0 +1,34 @@
+const express = require ('express');
+const server = express();
+const mongoose = require('mongoose');
+const User = require('./db/UserModel.js');
+
+

```

Comment:
we have to add cors support in this file . it's fairly easy but if you just add a comment it will be done by you or me sooner, otherwise frontent won't be able to connect to our server